### PR TITLE
Added basic unit tests for app credentials

### PR DESCRIPTION
**Pull request by isaiahwilliams**
# Description

With this pull request I am making the following changes

- Added basic unit testing for the ApplicationPartnerCredential class because of recent issues discovered with that class
- Replaced the new Integer ( string ) operations because they are considered depcreated

### CODE REVIEW

**Comment by izzyacademy** [comment]
In general, a lot of string constants are being repeated multiple times. It is best practice to have these strings defined as constants and the constants can be used instead. This makes it easy to maintain in the event a massive update is necessary in the future.

**Comment by isaiahwilliams** [comment]
Hi @izzyacademy, you are absolutely correct. I have made a small update to incorporate the changes you recommended. What do you think?

File: test/main/java/com/microsoft/store/partnercenter/TestConstants.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// -----------------------------------------------------------------------
+
+package com.microsoft.store.partnercenter;
+
+public class TestConstants 
+{
+    /**
+     * The Active Directory authority endpoint address.
+     */
+    public static final String AadAuthorityEndpoint = "https://login.microsoftonline.com";
```
Comment (msize) [inline test/main/java/com/microsoft/store/partnercenter/TestConstants.java]: The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_").

https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html

File: pom.xml [hunk 2]
```diff
@@ -5,103 +5,112 @@
...
+    </dependency>
+  </dependencies>
+  <build>
+    <sourceDirectory>src/main/java</sourceDirectory>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+      </resource>
+    </resources>
+    <testSourceDirectory>test/main/java</testSourceDirectory>
```
Comment (msize) [inline pom.xml]: The project should contain only one `main`. Commonplace for tests is `src/test/java`.

File: test/main/java/com/microsoft/store/partnercenter/TestConstants.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// -----------------------------------------------------------------------
+// <copyright file="TestConstants.java" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+package com.microsoft.store.partnercenter;
+
+public class TestConstants
```
Comment (msize) [inline test/main/java/com/microsoft/store/partnercenter/TestConstants.java]: Classes have all fields declared static. Giving such classes a private constructor prevents them from being inadvertently instantiated.

**Comment by isaiahwilliams** [comment]
Closing this PR because these changes will be incorporated into another request.

**Pull request closed without merging**